apps/elasticsearch/templates/services/app/dc.yml.j2

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: elasticsearch
+    service: elasticsearch
+    version: "{{ elasticsearch_image_tag }}"
+    type: es-node
+  name: "elasticsearch-node"
+  namespace: "{{ project_name }}"
+spec:
+  replicas: {{ elasticsearch_nodes }}
+  template:
+    metadata:
+      labels:
+        app: elasticsearch
+        service: elasticsearch
+        type: es-node
+    spec:
+      strategy:
+        type: Rolling
+        rollingParams:
+          maxUnavailable: 0
+          maxSurge: 1
+          timeoutSeconds: 3600
+      # Prefer running pods on different nodes for redundancy
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                    - key: deploymentconfig
+                      operator: In
+                      values:
+                        - "elasticsearch-node"
+                topologyKey: kubernetes.io/hostname
+      containers:
+        - image: {{ elasticsearch_image_name }}:{{ elasticsearch_image_tag }}
+          name: elasticsearch
+          ports:
+            - containerPort: {{ elasticsearch_api_port }}
+              protocol: TCP
+              name: "http-port"
+            - containerPort: {{ elasticsearch_communication_port }}
+              protocol: TCP
+              name: "comm-port"
```

Comment:
```suggestion
              name: "communication-port"
```

Reply:
My first attempt for this name was your suggestion... but :
```Invalid value: \\\\\"communication-port\\\\\": must be no more than 15 characters\"```

Reply:
:man_facepalming: lol